src/game/components/owned_expire_time.rs

```diff
@@ -1,10 +1,10 @@
 use std::time::Instant;
 
-pub struct ExpireTime {
+pub struct OwnedExpireTime {
     pub when: Instant,
 }
 
-impl ExpireTime {
+impl OwnedExpireTime {
```

Comment:
OwnerExpireTime instead of Owned.

Just because the expire should remove the Owner component from an entity.